typings/index.d.ts

```diff
@@ -1823,20 +1823,24 @@ export class Typing extends Base {
 
 export class User extends PartialTextBasedChannel(Base) {
   public constructor(client: Client, data: RawUserData);
+  public accentColor: number | null;
   public avatar: string | null;
+  public banner: string | null;
   public bot: boolean;
   public readonly createdAt: Date;
   public readonly createdTimestamp: number;
   public discriminator: string;
   public readonly defaultAvatarURL: string;
   public readonly dmChannel: DMChannel | null;
   public flags: Readonly<UserFlags> | null;
+  public readonly hexAccentColor: string | null;
```

Comment:
To make this compatible with ColorResolvable
```suggestion
  public readonly hexAccentColor: HexColorString | null;
```